molo/core/admin_views.py

```diff
@@ -15,8 +15,8 @@ def get(self, request, *args, **kwargs):
             page=article).reaction_question
         data_headings = ['Article']
         data_rows = []
-        choices = question.get_children().filter(
-            languages__language__is_main_language=True)
+        choices = ReactionQuestionChoice.objects.child_of(question).filter(
```

Comment:
`get_children()` won't give back the specific instances of the children. But since we know they are all `ReactionQuestionChoice` objects, we can query it in the reverse. We need the specific objects because the page objects do not have language on them